pyproject.toml

```diff
@@ -25,7 +25,6 @@ dependencies = [
     "pyyaml",
     "tqdm",
     "numpy",
-    "fsspec <= 2024.2.0", # Remove when pyarrow updates to reflect api changes
```

Comment:
Actually, is this meant to be removed completely? Do we still want an fsspec dependency?

Reply:
`hipscat` contains the fsspec dependency, and hipscat-import shouldn't rely on any fsspec-specific functionality.